src/_how_tos/sw-cli.md

```diff
@@ -68,7 +69,7 @@ For those who are curious, let's look at what's in the
 generated service worker.
 
 ```
-importScripts('sw-lib.v0.0.13.min.js');
+importScripts('workbox-sw.prod.v0.0.2.js');
```

Comment:
We can probably find a way to do this (Or at least make it trivial to update). Will look into this ASAP.